src/services/services.ts

```diff
@@ -3798,6 +3798,10 @@ module ts {
                 // Try getting just type at this position and show
                 switch (node.kind) {
                     case SyntaxKind.Identifier:
+                        if(isLabelName(node)){

```

Comment:
Fair enough; @GuillaumeSalles, you should move this check towards the bottom of `getTypeOfNode` in `checker.ts`.